### DISCUSSION TITLE
http请求被block导致按钮、数学公式等无法正常使用

### DISCUSSION THREAD

**Discussion by hydrogenhy** [hexojs/hexo#5522] [Question / Answer] [answered]
近期突然数学公式、按钮等无法生效，F12后查看报错为：
Mixed Content: The page at '<URL>' was loaded over HTTPS, but requested an insecure script 'http://cdn.staticfile.net/pace/1.2.4/pace.min.js'. This request has been blocked; the content must be served over HTTPS.

**Comment by SukkaW** [comment, marked as answer]
第三方公共 CDN 和 Hexo 无关，请自行关闭广告屏蔽插件。
